### ISSUE TITLE
clang-tidy-sarif 0.3.2 panics

### ISSUE THREAD

**Issue by igrr** [psastras/sarif-rs#283]
The recently released clang-tidy-sarif v0.3.2 panics on parsing the command line arguments.
clang-tidy-sarif v0.3.1 works correctly.

```bash
$ touch in.txt
$ ./clang-tidy-sarif in.txt
thread 'main' panicked at 'Must use `Arg::allow_invalid_utf8` with `_os` lookups at `[hash: 912D1D99AB2F394C]`', /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-3.2.15/src/parser/matches/arg_matches.rs:1732:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

This seems to be related to a change from clap 3.2.0 (https://github.com/clap-rs/clap/blob/master/CHANGELOG.md#320---2022-06-13)

I'll try to send a PR with the fix once I get some time.

**Comment by psastras** [comment]
@igrr thanks for the report -- i believe https://github.com/psastras/sarif-rs/pull/284 should fix this which will be bundled in 0.3.3

**Comment by psastras** [comment]
@igrr 0.3.3 is released with the fix. please try it and let me know if it doesn't fix the issue for you.

**Comment by igrr** [comment]
Thanks for the very quick fix, I can confirm that 0.3.3 works fine!

and thanks for developing and maintaining this very useful project!
